### PR TITLE
Added Gemini 😺

### PR DESCRIPTION
**Pull request by RAravindDS**
I like to add GEMINI in the list :)

### CODE REVIEW

File: src/llm/llm.py
Comment (nafSadh) [inline src/llm/llm.py:1]: gemini-pro is a deprecated name (it just points to Gemini 1.0 pro model)
you may also want to add gemini-1.5-pro-latest

**Comment by nafSadh** [comment]
Thanks for adding this.

File: src/llm/llm.py
Comment (RAravindDS) [reply src/llm/llm.py:1]: Yesterday when I was checking with Gemini available models list, it was available and working, If you want I can remove.

Comment (nafSadh) [reply src/llm/llm.py:1]: I think 1.5 is in preview.

Gemini-pro acts as an alias but it's confusing.

Comment (RAravindDS) [reply src/llm/llm.py:1]: Yeah, that's why I removed that

**Comment by mufeedvh** [comment]
This implementation exists in #24, will be reviewing that PR and comparing both. Thank you! ❤️ 🙌

**Pull request closed without merging**